tests/unittest/_torch/speculative/test_eagle3.py

```
@@ -366,11 +366,13 @@ def test_llama_eagle3_long_prompt(use_cuda_graph):
     else:
         cuda_graph_config = None
 
+    kv_cache_config = KvCacheConfig(free_gpu_memory_fraction=0.5)
```

Ruff (0.14.14)

[error] Local variable `kv_cache_config` is assigned to but never used

Remove assignment to unused variable `kv_cache_config`

(F841)